File: avaliacaoPPGI-spring-boot/src/main/java/avaliacaoPPGI/Conferencia.java
package avaliacaoPPGI;

class Conferencia extends Veiculo {

	public Conferencia(String sigla, String nome, float fatorDeImpacto) {
		super(sigla, nome, fatorDeImpacto);
	}

	@Override
	public String toString() {
		return "Conferencia [sigla=" + getSigla() + ", nome=" + getNome()
				+ ", fatorDeImpacto=" + getFatorDeImpacto() + "]";
	}

}

File: julia/cod/Conferencia.jl
using Base
include("Veiculo.jl")

struct Conferencia <: Veiculo
    @atributos_Veiculo

    Conferencia(sigla::String, nome::String, impacto::Float64) = 
        new(sigla, nome, impacto, Tuple{Int64, String}[])
end